src/Renraku-Tests/ReMethodSourceCleanerTest.class.st

```diff
@@ -7,8 +7,117 @@ Class {
 	#category : #'Renraku-Tests-Utilities'
 }
 
-{ #category : #tests }
+{ #category : #'private - test data' }
+ReMethodSourceCleanerTest >> aleadyCleanedMethodStringKeepingDot [
+	^'dirtyMethodWithUnnecessarySpacesAndFinalDot
+	"Do not format this method - it by intention should include unnecessary spaces and a final dot"
+
+	|temp|
+
+	temp := 3 + 4.'
+]
+
+{ #category : #'private - test data' }
+ReMethodSourceCleanerTest >> alreadyCleanedMethodString [
+	^'dirtyMethodWithUnnecessarySpacesAndFinalDot
+	"Do not format this method - it by intention should include unnecessary spaces and a final dot"
+
+	|temp|
+
+	temp := 3 + 4'
+]
```

Comment:
It looks like you have this method duplicated

Reply:
No - as you see and as the selector should reveal: one returns the cleand method with a dot and one without. It's also not an own method body - it returns a string we compare agains the #dirtyMethodWithUnnecessarySpacesAndFinalDot in the tests

Reply:
But I added comment in https://github.com/astares/pharo/commit/4904ac8610c2f07e369be13a25efb174b07400ba
too

Reply:
Didn't see the dot at the end :)

Reply:
That's why ideally there should be none :)